src/kvirc/ui/KviWindow.cpp

```diff
@@ -320,14 +320,13 @@ const char * KviWindow::m_typeTable[TypeCount] = {
 	"userwindow",   // 16
 	"tool",         // 17
 	"iograph",      // 18
-	"dirbrowser",   // 19
```

Comment:
I dont think its the same but at this stage Ive resigned to the fact we like trash so Ill comment them out